lib/kubernetes-deploy/render_task.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+require 'tempfile'
+
+require 'kubernetes-deploy/renderer'
+
+module KubernetesDeploy
+  class RenderTask
+    def initialize(logger:, current_sha:, template_dir:, bindings:)
+      @logger = logger
+      @template_dir = template_dir
+      @renderer = KubernetesDeploy::Renderer.new(
+        current_sha: current_sha,
+        bindings: bindings,
+        template_dir: @template_dir,
+        logger: @logger,
+      )
+    end
+
+    def run(*args)
+      run!(*args)
+      true
+    rescue KubernetesDeploy::InvalidTemplateError, KubernetesDeploy::TaskConfigurationError
+      false
+    end
+
+    def run!(filename, stream)
```

Comment:
What do you think about adding a bit more logging to make the UX of this command more similar to the others? Example:

![image](https://user-images.githubusercontent.com/4789493/48032684-96cfae80-e126-11e8-9b31-1421ac331c10.png)

Not a requirement for this PR, but I think it would be great if this ultimately gets an option to set the output dir to a file or maybe a directory instead of stdout if you want.

Reply:
Added some success logging